inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_binary_elementwise.cpp

```diff
@@ -334,6 +334,7 @@ INSTANTIATE_TEST_CASE_P(EltwiseBroadcastSingleDSR, DynamicToStaticShapeEltwiseSi
         ngraph::opset3::Add::type_info,
         ngraph::opset3::Divide::type_info,
 //        ngraph::opset3::Equal::type_info, operation broadcast default value needs to be fixed
+//        ngraph::opset3::Greater::type_info, operation broadcast default value needs to be fixed
```

Comment:
Do I understand correctly that there is an issue with `Broadcast` operation that prevents us from enabling these tests for `Equal` and `Greater`? If so, I think we should create a ticket on that (if there is no such yet) and mention its number there. So one who is fixing *broadcast issue* will see linked ticket (which probably is assigned to the same person) on enabling tests and will enable it once *broadcast issue* is resolved.

Reply:
It is totally another issue. Equal and Greater and other comparison operations are BinaryElementwises and they have operation parameter -- autobroadcasting, which makes it clear what broadcasting rule to use in case operation inputs should be broadcasted. These operations are not compliant with specification and unfortunately we have no ticket for that (will fix this in a moment and create it myself). But it doesn't affect the flow of dynamic operations -- only on-the-fly nodes creation inside of the tests is affected. These operations are supported from DTS perspective.